packages/brick_build/pubspec.yaml

```
@@ -4,21 +4,21 @@ homepage: https://github.com/GetDutchie/brick/tree/main/packages/brick_build
 issue_tracker: https://github.com/GetDutchie/brick/issues
 repository: https://github.com/GetDutchie/brick
 
-version: 4.0.0
+version: 4.1.0
 
 environment:
   sdk: ">=3.4.0 <4.0.0"
 
 dependencies:
-  analyzer: ">=6.11.0 <8.0.0"
+  analyzer: ">=6.11.0 <9.0.0"
```

Is this minimum version still valid? 
```suggestion
  analyzer: ">=8.0.0 <9.0.0"
```

---

Upgraded minimum